_posts/developers/apps/inrupt-tutorial/2020-11-06-00_getting-started.md

```diff
@@ -112,398 +108,393 @@ Node.js installation.
-        
-                <div id="read" class="panel">
-                    <div class="row">
-                        <label id="writelabel" for="name">2. Write your name: </label>
-                        <input
-                                type="text"
-                                id="input_name"
-                                name="name"
-                                size="50"
-                                value="Your name here"
-                        />
-                        <button name="btnWrite" id="btnWrite">Write to Profile</button>
-                    </div>
-        
-                    <dl class="display">
-                        <dt>Writing status:&nbsp</dt>
-                        <strong><span id="labelWriteStatus">...not written yet...</span></strong>
-                    </dl>
+1. Create an `index.html` file with the following content:
+
+    ```html
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <meta charset="utf-8" />
+            <title>Getting Started: Inrupt JavaScript Client Libraries</title>
+            <script defer src="./index.js"></script>
+            <link rel="stylesheet" href="my-demo.css" />
+        </head>
+    
+        <body>
+            <header>
+                <h2>Getting Started</h2>
+                <h3>with Inrupt JavaScript Client Libraries</h3>
+            </header>
+            <section id="login" class="panel">
+                <div class="row">
+                    <label id="labelLogin" for="btnLogin">1. Click the button to log into
+                        <span id="solid_identity_provider">...provided by the JavaScript code...</span>:
+                    </label>
+                    <button name="btnLogin" id="btnLogin">Login</button>
+                    <p id="labelStatus"></p>
                 </div>
-        
-                <div id="read" class="panel">
-                    <div class="row">
-                        <label id="readlabel" for="webID"
-                        >3. Read back name (anyone's!) from their WebID:
-                        </label>
-                        <input
-                                type="url"
-                                id="webID"
-                                name="webID"
-                                size="50"
-                                value="...not logged in yet - but enter any WebID to read from its profile..."
-                        />
-                        <button name="btnRead" id="btnRead">Read Profile</button>
-                    </div>
-                    <dl class="display">
-                        <dt>Formatted Name (FN) read from Pod:&nbsp</dt>
-                        <strong><span id="labelFN">...not read yet...</span></strong>
-                    </dl>
+            </section>
+    
+            <div id="read" class="panel">
+                <div class="row">
+                    <label id="writelabel" for="name">2. Write your name: </label>
```

Comment:
Given that later in the Run section of the tutorial we make people do the Read Profile first before we have them Write their name, we may want to switch these panel ordering in the html.